scripts.js

```diff
@@ -1,68 +1,82 @@
-var apiurl, myresult;
-
-headerurl = "https://api.flickr.com/services/rest/?method=flickr.photosets.getPhotos&api_key=bc1b5549bb490dcce777b6659b9740f7&photoset_id=72157655762735509&privacy_filter=1&per_page=1&format=json&nojsoncallback=1";
-
-apiurl =
-  "https://api.flickr.com/services/rest/?method=flickr.galleries.getPhotos&api_key=bc1b5549bb490dcce777b6659b9740f7&gallery_id=72157690043876864&per_page=8&format=json&nojsoncallback=1";
-
+var myResult;
+var headerUrl =
+  'https://api.flickr.com/services/rest/?method=flickr.photosets.getPhotos&api_key=bc1b5549bb490dcce777b6659b9740f7&photoset_id=72157655762735509&privacy_filter=1&per_page=1&format=json&nojsoncallback=1';
+var apiUrl =
+  'https://api.flickr.com/services/rest/?method=flickr.galleries.getPhotos&api_key=bc1b5549bb490dcce777b6659b9740f7&gallery_id=72157690043876864&per_page=8&format=json&nojsoncallback=1';
 
+$(window).on('load', function() {
+  AOS.init({
+    duration: 1200
+  });
 
-$(window).on('load', function () {	
-	AOS.init({
-  		duration: 1200,
-	})
+  $.getJSON(headerUrl, function(json) {
+    $.each(json.photoset.photo, function(i, myResult) {
+      var imgSrc =
+        'http://farm' +
```

Comment:
Breaking up this string concatentation like this is a Prettier default, but you can also do:

```js
var imgSrc =
  `http://farm#{myResult.farm}.static.flickr.com/${myResult.server}/${myResult.id}_${myResult.secret}_b.jpg`;
```